App crashes in background mode when enteredRegion fires

When the 'enteredRegion' fires when the app is in the background, the app crashes.

```
Jun  6 15:11:30 Andrew-Hawkinss-iPhone Mynt[1618] <Warning>: [INFO] Entered region Estimote
Jun  6 15:11:30 Andrew-Hawkinss-iPhone Mynt[1618] <Warning>: [INFO] Turning on ranging for (null)
Jun  6 15:11:30 Andrew-Hawkinss-iPhone Mynt[1618] <Warning>: [ERROR] The application has crashed with an uncaught exception 'NSInvalidArgumentException'.
Reason:
*** setObjectForKey: key cannot be nil
Stack trace:

0   CoreFoundation                      0x2d4e6f0b <redacted> + 130
1   libobjc.A.dylib                     0x37c7dce7 objc_exception_throw + 38
2   CoreFoundation                      0x2d4221bf <redacted> + 818
3   Mynt                                0x0027e2e3 _ZN3WTI13tryMakeStringIPKcN2TI7UStringES2_EENS_10PassRefPtrINS_10StringImplEEET_T0_T1_ + 3802
4   Mynt                                0x0027e03b _ZN3WTI13tryMakeStringIPKcN2TI7UStringES2_EENS_10PassRefPtrINS_10StringImplEEET_T0_T1_ + 3122
5   CoreLocation                        0x2d9ab4ab CLLocationCoordinate2DGetDistanceFrom + 21918
6   CoreLocation                        0x2d9a6aeb CLLocationCoordinate2DGetDistanceFrom + 3038
7   CoreLocation                        0x2d9a0081 CLClientInvalidate + 944
```

Code:

```
TiBeacons.addEventListener("enteredRegion", enterRegion);
function enterRegion(e) {
    Ti.API.info(e);
    notify("enterRegion: "+e.identifier, e);
}
```

Ti SDK is 3.2.3 GA and iOS is 7.1.1

--- a/Classes/OrgBeuckmanTibeaconsModule.m
+++ b/Classes/OrgBeuckmanTibeaconsModule.m
@@ -237,7 +237,12 @@
                            nil];
     
     if (_autoRange) {
-        [self turnOnRangingWithRegion:[self.monitoringRegions objectForKey:region.identifier]];
+        CLBeaconRegion *beaconRegion = [self.monitoringRegions objectForKey:region.identifier];
+        if (beaconRegion) {
+            [self turnOnRangingWithRegion:beaconRegion];
+        } else {
+            NSLog(@"[INFO] Cannot find beaconRegion to range");
+        }
     }
     
     [self fireEvent:@"enteredRegion" withObject:event];
@@ -293,6 +298,11 @@
     
     if (![CLLocationManager isRangingAvailable]) {
         NSLog(@"[INFO] Couldn't turn on ranging: Ranging is not available.");
+        return;
+    }
+    
+    if (beaconRegion == nil) {
+        NSLog(@"[ERROR] Unable to find beaconRegion for beaconRegion");
         return;
     }
     

